crates/pink/pink-extension/Cargo.toml

```diff
@@ -24,4 +25,5 @@ std = [
     "scale-info/std",
 ]
 runtime_utils = ["std"]
+dlmalloc = ["ink/no-allocator", "dep:dlmalloc"]
```

Comment:
Is `dep:dlmalloc` the new notation in cargo to explicitly target the dependency?

Reply:
Yes. It was released about a year ago.